test/system/src/test/java/io/pravega/test/system/DynamicRestApiTest.java

```diff
@@ -0,0 +1,139 @@
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.glassfish.jersey.client.authentication.HttpAuthenticationFeature;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import mesosphere.marathon.client.MarathonException;
+
+import static org.junit.Assert.assertEquals;
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+@Slf4j
+@RunWith(SystemTestRunner.class)
+public class DynamicRestApiTest extends AbstractSystemTest {
+
+    @Rule
+    public Timeout globalTimeout = Timeout.seconds(5 * 60);
+
+    private final Client client;
+    private WebTarget webTarget;
+    private String restServerURI;
+    private String resourceURl;
+
+    public DynamicRestApiTest() {
+
+        org.glassfish.jersey.client.ClientConfig clientConfig = new org.glassfish.jersey.client.ClientConfig();
+        clientConfig.register(JacksonJsonProvider.class);
+        clientConfig.property("sun.net.http.allowRestrictedHeaders", "true");
+
+        if (Utils.AUTH_ENABLED) {
+            HttpAuthenticationFeature feature = HttpAuthenticationFeature.basic("admin", "1111_aaaa");
+            clientConfig.register(feature);
+        }
+
+        client = ClientBuilder.newClient(clientConfig);
+    }
+
+    /**
+     * This is used to setup the various services required by the system test framework.
+     *
+     * @throws InterruptedException If interrupted
+     * @throws MarathonException    when error in setup
+     * @throws URISyntaxException   If URI is invalid
+     */
+    @Environment
+    public static void initialize() throws MarathonException {
+        URI zkUri = startZookeeperInstance();
+        startBookkeeperInstances(zkUri);
+        URI controllerUri = ensureControllerRunning(zkUri);
+        ensureSegmentStoreRunning(zkUri, controllerUri);
+    }
+
+    @Test
+    public void listScopes() {
+        Service conService = Utils.createPravegaControllerService(null);
+        List<URI> ctlURIs = conService.getServiceDetails();
+        URI controllerRESTUri = ctlURIs.get(1);
+        Invocation.Builder builder;
+        Response response;
+        String protocol = Utils.TLS_AND_AUTH_ENABLED ? "https://" : "http://";
+        restServerURI = protocol + controllerRESTUri.getHost() + ":" + controllerRESTUri.getPort();
+        log.info("REST Server URI: {}", restServerURI);
+
+        assertFalse(true);
+
+        // TEST REST server status, ping test
+        resourceURl = new StringBuilder(restServerURI).append("/ping").toString();
+        webTarget = client.target(resourceURl);
+        builder = webTarget.request();
+        response = builder.get();
+        assertEquals("Ping test", OK.getStatusCode(), response.getStatus());
+        log.info("REST Server is running. Ping successful.");
```

Comment:
This log statement is redundant, the control'd only move forward if the previous assertion passes. Regardless of whether the next assertion succeeds or fails, one'd know that the ping was successful. 

Having said that, it's alright if you want to keep it anyway.